rust/src/crypto.rs

```diff
@@ -761,7 +761,53 @@ impl_hash_type!(BlockHash, 32);
 impl_hash_type!(VRFVKey, 32);
 impl_hash_type!(KESVKey, 32);
 // same for this signature
-impl_hash_type!(KESSignature, 448);
+//impl_hash_type!(KESSignature, 448);
+// TODO: when >32 size trait implementations are out of nightly and into stable
```

Comment:
Note that this can take some time, since even in the current stable some tier-3 components are still missing is targets `armv7-apple-ios` and `i386-apple-ios`